### PR TITLE
Attempting to add useful info, mostly specific to Monero.

### PR DESCRIPTION
**Pull request by apezio**
Havent tested the changes on a live version of Moneropedia.

### CODE REVIEW

**Comment by apezio** [comment]
Deleted easiest_buy (Circle no longer sells Bitcoin) and wrote a new guide for buying Monero via Shapeshift and securely storing it on a paper wallet.  Pics to come later.

Reorganized the index and added the guide -^

**Comment by fluffypony** [comment]
Please fix conflicts and squash commits:)

File: knowledge-base/moneropedia/ringCT.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-
+---
+layout: moneropedia
+entry: "Ring CT"
+terms: ["ringCT", "ring-CT"]
+summary: "a way to hide the amount sent in a Monero transaction"
+---
+
+### The Basics
+<<<<<<< master
```
Comment (erciccione) [inline knowledge-base/moneropedia/ringCT.md]: You forgot a piece: <<<<<<< master
;)

Comment (erciccione) [inline knowledge-base/moneropedia/ringCT.md]: You forgot a piece: <<<<<<< master
;)

**Comment by apezio** [comment]
Thank you!  I've never used github before. :)

**Comment by erciccione** [comment]
No problem @apezio , but now you have to squash all those commits in one, or at least fewer. Did you use only github or you have a local repository managed with git? because if you are using just github i have no idea if it's possible to squash the commits. If you are using git,[ this link](http://www.eqqon.com/index.php/Collaborative_Github_Workflow) helped me (i started with git and github just few weeks ago). Hope this will help

**Pull request closed without merging**